src/api/routes/rosetta/construction.ts

```diff
@@ -231,8 +229,7 @@ export function createRosettaConstructionRouter(db: DataStore, chainId: ChainID)
             noneCV(),
             noneCV(),
           ],
-          validateWithAbi: true,
-          network: getStacksNetwork(),
+          validateWithAbi: false,
```

Comment:
Think we need to add these to the offline-rosetta tests? Like I don't know off the top of my head of if `makeUnsignedContractCall` works without network when the `fee`/`nonce` are unspecified

Reply:
You're right,  It won't work I double-checked and It doesn't work without fee and network. 
So, I think I should provide a hardcoded fee here as well. 

Thanks for pointing that out.

Reply:
Fix looks good. We can merge if you've manually tested and verified the fix, but we should try to get this in the offline rosetta tests at some point.

Reply:
I have manually tested it with offline mode.
We do have a test for `/preprocess`, I don't know how to test this behaviour though. one idea that comes to mind is that we can disallow all network calls with `nock` but I don't know if it will work or not.

Reply:
@zone117x It works!. checkout the new commit!